redisinsight/api/src/modules/cloud/subscription/models/api.interface.ts

```diff
@@ -0,0 +1,21 @@
+import { CloudSubscriptionPlan } from './cloud-subscription-plan';
+import { CloudSubscriptionRegion } from './cloud-subscription-region';
+
+export class CloudSubscriptionPlanResponse extends CloudSubscriptionPlan {
```

Comment:
basically it is dto CloudSubscriptionPlanResponse. please, move to dto folder

Reply:
moved